src/code/z_actor.c

```diff
@@ -2472,7 +2454,10 @@ void func_800315AC(GlobalContext* globalCtx, ActorContext* actorCtx) {
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_actor.c", 6563);
 }
 
-void func_80031A28(GlobalContext* globalCtx, ActorContext* actorCtx) {
+/**
+ * Kills all actors in ActorContext whose object dependency is not loaded.
+ */
+void Actor_KillIfObjectIsNotLoaded(GlobalContext* globalCtx, ActorContext* actorCtx) {
```

Comment:
Wording is weird here, but I don't have any suggestions.

Reply:
yeah I had trouble with this one too. I wanted to name it something like `Actor_KillAllObjectsInContextIfTheirObjectDependencyIsNotLoaded` but that doesn't work well for obvious reasons.

Reply:
`Actor_KillActorsMissingObjects`?

Reply:
My only issue with that is the redundancy of the word "actor" twice, but overall I don't mind that name.

Reply:
`KillIfObjectMissing`?

Reply:
maybe `Actor_KillAllIfObjectMissing`?